pages/en/tokenomics.mdx

```diff
@@ -0,0 +1,102 @@
+
+By using The Graph, users can easily access data from the blockchain, paying only for the specific information they need. The Graph is used by many popular applications in the web3 ecosystem today.
+
+The Graph indexes blockchain data similarly to how Google indexes the web. In fact, you may already be using The Graph without realizing it. If you've viewed the front end of a dapp that gets its data from a subgraph, you queried data from a subgraph!
+
+The Graph plays a crucial role in making blockchain data more accessible and enabling a marketplace for its exchange.
+
+# The Roles of Network Participants
+
+There are four primary network participants:
+
+1.  Delegators - Delegate GRT to Indexers & secure the network
+
+2.  Curators - Find the best subgraphs for Indexers
+
+3.  Developers - Build & query subgraphs
+
+4.  Indexers - Backbone of blockchain data
+
+Fishermen and Arbitrators are also integral to the network's success through other contributions, supporting the work of the other four. For more information about network roles, [read this article](https://thegraph.com/blog/the-graph-grt-token-economics/).
+
+![Tokenomics diagram](/img/Tokeneconomics-Diagram@2x.png)
+
+# Delegators (Passively earn GRT)
+
+Delegators delegate their GRT to Indexers, which helps Indexers increase their stake in subgraphs on the network. In return, Delegators earn a percentage of all query fees and indexing rewards from the Indexer. Each Indexer sets the cut that will be rewarded to Delegators independently, creating competition among Indexers to attract Delegators. Most Indexers offer between 9-12% annually.
+
+For example, if a Delegator were to delegate 15k GRT to an Indexer offering 10%, the Delegator would receive ~1500 GRT in rewards annually.
+
+There is a 0.5% delegation tax which is burned whenever a Delegator delegates GRT on the network. If a Delegator chooses to withdraw their delegated GRT, the Delegator must wait for the 28 epoch unbonding period. Each epoch is 6,646 blocks, which means 28 epochs ends up being approximately 26 days.
+
+If you're reading this, you're capable of becoming a Delegator right now by heading to the [network participants page](https://thegraph.com/explorer/participants/indexers), and delegating GRT to an Indexer of your choice.
+
+# Curators (Earn GRT)
+
+Curators identify high-quality subgraphs, and "curate" them (i.e., signaling GRT on them) to earn curation shares, which guarantee a percentage of all future query fees generated by the subgraph. While any independent network participant can be a Curator, typically subgraph developers are among the first Curators for their own subgraphs because they want to ensure their subgraph is indexed.
```

Comment:
```suggestion
Curators identify high-quality subgraphs, and "curate" them (i.e., signal GRT on them) to earn curation shares, which guarantee a percentage of all future query fees generated by the subgraph. While any independent network participant can be a Curator, typically subgraph developers are among the first Curators for their own subgraphs because they want to ensure their subgraph is indexed.
```